src/app/auth/pages/logout/logout.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from '@app/auth/services';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-logout',
+  templateUrl: './logout.component.html',
+  styleUrls: ['./logout.component.scss']
+})
+export class LogoutComponent implements OnInit {
+
+  constructor(private authenticationService: AuthService,
+    private router: Router) { }
+
+  ngOnInit() {
+    this.authenticationService.logout();
```

Comment:
It's better to create this component like a dumb and move logout logic after button "logout" is clicked.

Reply:
Remove this component, refactoring code